libs/base/ui/overlays/modal/src/stories/modal-demo.stories.ts

```diff
@@ -26,6 +26,9 @@ interface Props {
 
   firstModalPreventCloseByEscape?: boolean;
   secondModalPreventCloseByEscape?: boolean;
+
+  firstFooterButtonFullWidth?: boolean;
+  secondFooterButtonFullWidth?: boolean;
```

Comment:
Can you add static stories with full width button?

Reply:
Forgot to add the file. It's there now.